graylog2-web-interface/src/components/indices/IndexSetsComponent.jsx

```diff
@@ -64,7 +66,14 @@ const IndexSetsComponent = createReactClass({
 
   _onSetDefault(indexSet) {
     return () => {
-      IndexSetsActions.setDefault(indexSet).then(() => this.loadData(this.currentPageNo, this.currentPageSize));
+      IndexSetsActions.setDefault(indexSet).then(() => {
+        this.loadData(this.currentPageNo, this.currentPageSize);
```

Comment:
I think we should not make tracking dependent on any calls or actions. If I understand correctly, we want to track user behavior, so I'd track on click immediately, as the user clicks anyways, no matter if calls fail or not.